loadable_apps/micomapp/Makefile

```diff
@@ -136,11 +135,11 @@ endif
 	$(Q) $(TOPDIR)/tools/mkchecksum.py $(USER_BIN_DIR)/$(BIN)
 
 verify:
-ifneq ($(BIN_CHECK),0)
-	@echo "Undefined Symbols"
-	$(Q) $(NM) -u -l $(BIN)
-	$(call DELFILE, $(SYSTEM_BIN_DIR)/$(BIN))
-	$(call DELFILE, $(BIN))
-	$(call CLEAN)
-	exit 1
+ifeq ($(CONFIG_APP_BINARY_SEPARATION),y)
+	$(Q) if [ "`nm -u $(BIN) | wc -l`" != "0" ]; then \
+		echo "Undefined Symbols"; \
+		nm -u -l $(BIN); \
+		rm $(BIN); \
+		exit 1; \
+	fi
```

Comment:
Instead of adding this check block in every loadable app Makefile, Is it not good to put this block into a shell script inside tools directory and invoke from every loadable app Makefile?

Reply:
@manoharahk Hello. Yeah, as you told, it is moved into loadable.mk file. Please find the last commit.